components/observations/ObservationsListView.tsx

```diff
@@ -71,9 +74,28 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
   const originalFilterConfig: ObservationFilterConfig = useMemo(() => createDefaultFilterConfig(additionalFilters), [additionalFilters]);
   const [filterConfig, setFilterConfig] = useState<ObservationFilterConfig>(originalFilterConfig);
   const [filterModalVisible, {set: setFilterModalVisible, on: showFilterModal}] = useToggle(false);
+  const {data: metaData} = useAvalancheCenterMetadata(center_id);
+  const alternateZonesUrl = metaData?.widget_config?.observation_viewer?.alternate_zones || '';
+  const {data: alternateZones} = useAlternateObservationZones(alternateZonesUrl, {
+    enabled: !!alternateZonesUrl,
+  });
+
   const mapResult = useMapLayer(center_id);
   const mapLayer = mapResult.data;
 
+  const mergedMapLayer = useMemo(() => {
+    if (alternateZones && mapLayer) {
+      const zonesNotInMapLayer: MapLayerFeature[] = (alternateZones as unknown as MapLayerFeature[]).filter(
+        (zone: MapLayerFeature) => !mapLayer.features.some((feature: MapLayerFeature) => feature.properties.name === zone.properties.name),
+      );
+      return {
+        ...mapLayer,
+        features: [...mapLayer.features, ...zonesNotInMapLayer],
```

Comment:
When we display observations to the user in card form, are we grabbing the first item from this list that matches for labelling? I think we should try to be explicit - do users expect to see an observation matching a custom zone and a forecast zone labelled with the former or the latter?

Reply:
@stevekuznetsov, sorry, I'm not entirely sure what you mean here. Are you saying that we want to make sure that if there are multiple matches, we want to be explicit about prioritizing the forecast zone?

Reply:
I'm actually not sure what avalanche center staff would expect in this situation - I've reached out to the NAC and I'll follow up here once I hear back. However, regardless of which path is taken, I was hoping we could have explicit logic `(observation, forecast_zones, custom_zones) => label` that would encode the choice, as opposed to having implicit logic (return the first match from a list containing both types of zones that might or might not be sorted in a specific order).

Reply:
Got some clarity from the NAC, we should prioritize the custom zones if present.

Reply:
Thanks for getting clarification on that!